builtin/providers/aws/resource_aws_elasticache_cluster.go

```diff
@@ -72,6 +72,11 @@ func resourceAwsElasticacheCluster() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
+			"replication_group_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
```

Comment:
This seems like a `Computed` attribute to me, what do you think? It seems we would have a `aws_elasticache_cluster` resource, and then a `elasticache_replication_group` which has a `primary_cluster_id` that points to the `aws_elasticache_cluster`. The `aws_elasticache_cluster` can exist by itself, but it may have a replication group associated with it, which comes later. Does that make sense?